content/cloud-computing/cloud-computing.md

```diff
@@ -0,0 +1,22 @@
+---
+Title: 'Cloud Computing'
+Description: 'Cloud computing refers to an architecture where computing resources and hardware are maintained and managed remotely by an entity other than the user.'
+Codecademy Hub Page: 'https://www.codecademy.com/catalog/subject/cloud-computing'
+CatalogContent:
+  - 'learn/foundations-of-cloud-computing'
+  - 'paths/back-end-engineer-career-path'
+---
+
+**Cloud computing** refers to an architecture where computing resources and hardware are maintained and managed remotely by an entity other than the user. This is becoming popular with businesses and ubiquitous in personal computing. For example, whenever a person sets up a website on GoDaddy, plays World of Warcraft, or streams a movie on Netflix, they are using a cloud computing solution.
+
+## Advantages
+
+Cloud computing offers the following advantages to the user:
+
+- **Scalability:** Adding additional capacity is quicker and easier in a cloud environment then in an on-premises data center. When a user manages a data center themselves, additional capacity involves purchasing, installing, and configuring new hardware. A cloud solution provides access to many more resources than would otherwise be possible.
+- **Security:** A cloud solution often provides more security than a business, especially a small business, can manage on their own.
+- **Availability:** Most cloud solutions have more robust and redundant architecture than a small to medium business can afford, leading to better uptime for users and clients.
+- **Performance:** A cloud solution will provide more powerful hardware than a business could afford in its own data center.
+- **Cost:** A cloud solution is typically more affordable than an on-premises solution offering the same capabilities.
+
+Generally, cloud computing removes the necessity of a business to maintain a dedicated data center, with all the hardware and personnel it requires, for a particular solution, allowing the business to allocate those resources to other tasks.
```

Comment:
Run on sentence. Perhaps:

Cloud computing removes the need for a business to maintain a dedicated data center. This frees the business from the cost of all the personnel and hardware that a data center requires.